extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/CompatParquetReaderTest.java

```diff
@@ -90,20 +92,24 @@ public void testBinaryAsString() throws IOException
     );
     List<InputRowListPlusRawValues> sampled = sampleAllRows(reader);
     List<InputRowListPlusRawValues> sampledAsBinary = sampleAllRows(readerNotAsString);
+    ObjectMapper objectMapper = new ObjectMapper();
     final String expectedJson = "{\n"
                                 + "  \"field\" : \"hey this is &é(-è_çà)=^$ù*! Ω^^\",\n"
                                 + "  \"ts\" : 1471800234\n"
                                 + "}";
-    Assert.assertEquals(expectedJson, DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues()));
-
+    final String serializedJson = DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues());
```

Comment:
Could you please make the same change for all the asserts in this file?

Reply:
> Could you please make the same change for all the asserts in this file?

It seems to me that all the asserts in this file related to JSON string comparison have been fixed in this commit. Could you help point me out where are those ones remaining? Thank you!

Reply:
Hi! what I meant was instead of creating new variables, could you modify the code such that it's just one line of change? - this will help us in getting the PR merged faster. - you can refer to my suggestion here -https://github.com/yashdeep97/druid/pull/4#discussion_r1411223096

Reply:
> Hi! what I meant was instead of creating new variables, could you modify the code such that it's just one line of change? - this will help us in getting the PR merged faster. - you can refer to my suggestion here -[#4 (comment)](https://github.com/yashdeep97/druid/pull/4#discussion_r1411223096)

Though I don't mind changing it, since I'm not the one who raised this pr, can I still make changes to it? I just contributed one commit in this file and send a pr to @prathyushreddylpr .